### PR TITLE
faq: add section on running docker and podman at the same time

### PR DESCRIPTION
**Pull request by dustymabe**
See https://github.com/coreos/fedora-coreos-tracker/issues/408

### CODE REVIEW

**Comment by jlebon** [comment]
> Running containers via `docker` and `podman` at the same time can cause
issues and unexpected behavior.

Huh, I didn't realize this was an issue. Is this documented in more details somewhere (else)? I couldn't find anything in a quick search.

**Comment by dustymabe** [comment]
@miabbott noted that mentioning how we have docker configured (socket activation) would be useful for people trying to understand the problem we are trying to solve by masking the unit.

**Comment by dustymabe** [comment]
@miabbott one more force push to clean up some wording

File: modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -201,3 +201,35 @@ To learn more about these APIs, see the AWS documentation for
...
+
+No. Running containers via `docker` and `podman` at the same time can cause
+issues and unexpected behavior. We highly recommend against trying to use them
+both at the same time. 
+
+It is worth noting that in Fedora CoreOS we have `docker.service`
+disabled by default but it is easily started if anything communicates
+with the `/var/run/docker.sock` because `docker.socket` is enabled by
+default. This means that if a user runs any `docker` command (via
+`sudo docker`) then the daemon will be brought up. We did this to make
```
Comment (miabbott) [inline modules/ROOT/pages/faq.adoc]: `s/brought up/activated/` ?

Comment (dustymabe) [reply modules/ROOT/pages/faq.adoc]: updated

**Pull request merged**